src/pages/Congratulations/index.tsx

```diff
@@ -1,13 +1,21 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
```

Comment:
```suggestion
import React, { useEffect, useState } from 'react';
```
We still use these hooks